native/packages/react-native-bpk-component-text/src/common-types.js

```diff
@@ -87,6 +87,7 @@ export const weightPropType = (
   props: Props,
   propName: string,
   componentName: string,
+  ...rest: Array<any>
```

Comment:
React passes a secret argument to prop types to identify the source of the caller. We just need to spread the arguments through in custom validators.